test-utils/chain-spec-builder/Cargo.toml

```diff
@@ -5,7 +5,10 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-clap = { version = "~2.32.0", features = ["yaml"] }
+ansi_term = "0.12.1"
+clap = { version = "~2.32.0" }
```

Comment:
```suggestion
```

It is not used anymore? Or does structopt requires it?

Reply:
Not sure, need to check but probably not needed.